app/command/execute/main/mode/noEnv/config-helper.js

```diff
@@ -0,0 +1,28 @@
+path = require("path");
+const {currentDateWithTime} = require("../../../../../utils/date-utils");
+const {createDirectoryIfNotExist} = require("../../../../../utils/fs-helper");
+
+const PARAM_DELIMITER = "=";
+const TEMP_DIR = "/opt/sls-tools-temp";
+
+const processParamsForNoEnv = (configuration, cmdArgs) => {
+    if (cmdArgs.params) {
+        configuration.executeParams = cmdArgs.params.reduce((acc, parameterItem) => {
```

Comment:
tenhle kod je duplicita k tomu co už existuje v confiugration-reader, zkus se zamyslet jesrtli by to nešlo nejak přepoužít